template/addons/trpc/protected-example-router.ts

```diff
@@ -0,0 +1,14 @@
+import { createProtectedRouter } from "./protected-router";
+
+// Create queries & mutations with the surety that user has a session (due to createProtectedRouter).
```

Comment:
// Example router with queries that can only be hit if the user requesting is signed in